frontend/src/app/navbar/navbar.component.html

```diff
@@ -1,22 +1,10 @@
-<nav class="navbar navbar-light bg-light" style="margin-bottom: 50px;">
-  <a class="navbar-brand" href="#">
-    <img src="../../assets/logo.png" width="250" height="115" alt="Bulwark Logo" /></a>
-  <div class="mx-auto">
-    <div *ngIf="loaderService.isLoading | async" role="status">
-      <p-progressSpinner [style]="{width: '75px', height: '75px'}"></p-progressSpinner>
-    </div>
-  </div>
-  <div class="nav-item dropdown" *ngIf="loggedIn$ | async as loggedIn">
-    <a class="nav-link dropdown-toggle" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true"
-      aria-expanded="false">
-      Menu
+<div *ngIf="loaderService.isLoading | async" role="status">
+  <p-progressSpinner [style]="{width: '75px', height: '75px'}"></p-progressSpinner>
+</div>
+<p-menubar [model]="menuItems" *ngIf="loggedIn$ | async as loggedIn">
```

Comment:
Add space between the menu bar and the body web components.  Currently, they are attached.

<img width="738" alt="Screen Shot 2020-10-18 at 10 46 35 AM" src="https://user-images.githubusercontent.com/1582016/96371335-37121880-112f-11eb-8a9e-41f2e713cb1e.png">